### PR TITLE
save arrow format embeddings as-is

### PR DESCRIPTION
**Pull request by apage43**
only land once that's what's actually served here

### CODE REVIEW

File: nomic/project.py [hunk 1]
```diff
@@ -516,12 +519,12 @@ def download_shard(offset, check_access=False):
...
             try:
-                content = response.json()
+                content = response.content
 
-                shard_name = '{}_{}_{}.pkl'.format(self.atlas_index_id, offset, offset + limit)
+                shard_name = '{}_{}_{}.feather'.format(self.atlas_index_id, offset, offset + limit)
                 shard_path = os.path.join(save_directory, shard_name)
                 with open(shard_path, 'wb') as f:
-                    pickle.dump(content, f)
+                    f.write(content)
```
Comment (bmschmidt) [inline nomic/project.py:527]: Should we validate by reading the bytes into Arrow? If not might be worth checking that:
```
content[:6] == b"ARROW1" and content[-6:] == b"ARROW1"
```

Comment (bmschmidt) [reply nomic/project.py:527]: (But maybe we have a status==200 check above that guarantees this?)

Comment (apage43) [reply nomic/project.py:527]: wouldn't hurt to be sure the format is right and not just that the status is correct, especially since the API changed

**Comment by AndriyMulyar** [comment]
ci should be fixed now (was screwed up earlier)

**Comment by apage43** [comment]
CI didn't trigger this time either?

**Review by AndriyMulyar — APPROVED** [review]

**Pull request merged**